hangman.js

```diff
@@ -1 +1,78 @@
-console.log("welcome to hangman");
\ No newline at end of file
+
+
+var welcome = alert("Welcome to Hang My Best Friend! This is your host, Master Bipin and you get to hang your best friend in this show. Let's Begin!!");
+
+var name = prompt ("What is your name?");
+
+
+
+var word = prompt("Hi, " + name + "! You're a shitty friend for wanting to hang your best friend but that's NOM Biz. What's your word?");
+
+var hiddenword = word.split("");
+
+var mask  = word.replace(/./g, " _ ");
+
+
+var best = prompt(name +", which friend do you want to hang today?");
+
+
+
+
+var goodguess = []
+var badguess = []
+
+while (goodguess.length < word.length) {
+
+  var guessedLetter = prompt(mask + " We have a word! " + best + " Start guessing the word " + name + " chose for your death!")
+
+  var place = word.indexOf(guessedLetter);
```

Comment:
What if a letter appears more than once in a word? `indexOf` only returns the index of the argument's **first** instance in the word.